docs/general/ddl/constraints.rst

```diff
@@ -57,6 +57,27 @@ Or using a alternate syntax::
 
    For further details see :ref:`primary_key_constraint`.
 
+.. _constraints-null:
+
+null
+========
```

Comment:
```suggestion
NULL
====
```

Reply:
I didn't make it uppercase because of all the other headers, but make sense, so I'm changing everything to uppercase (Primary Key, Check, Not null)